pages/submission/PKPSubmissionHandler.php

```diff
@@ -158,6 +158,26 @@ protected function showWizard(array $args, Request $request, Submission $submiss
         /** @var Publication $publication */
         $publication = $submission->getCurrentPublication();
 
+        $section = Application::getSectionDAO()->getById($publication->getData(Application::getSectionIdPropName()), $context->getId());
+        if (
+            $section && 
+            (
+                $section->getIsInactive() || 
+                ($section->getEditorRestricted() && !$this->isEditor())
+            )
+        ) {
+            $this->showErrorPage(
+                'submission.wizard.sectionClosed',
+                __('submission.wizard.sectionClosed.message', [
+                    'contextName' => $context->getLocalizedData('name'),
+                    'section' => $section->getLocalizedTitle(),
+                    'email' => $context->getData('contactEmail'),
+                    'name' => $context->getData('contactName'),
```

Comment:
Perhaps these variables, which will be outputted as HTML to the frontend need some escaping.

Reply:
It's my understanding that we don't escape values from context settings (including section settings) because we trust the users who input them (`ROLE_ID_MANAGER`). It's worth bringing up again with Alec at a dev call, though, in case anything has changed there.